docs/windows_developer_guide.md

```diff
@@ -89,37 +77,18 @@ set HTTP_PROXY=my.proxy.com:123
 set HTTPS_PROXY=my.proxy.com:122
 ```
 
-## OPENCV install to - "C:\\opt\\"
-https://github.com/opencv/opencv/releases/download/4.10.0/opencv-4.10.0-windows.exe
-
-# OPENCV contrib for optflow
-```
-cd c:\opt
-git clone https://github.com/opencv/opencv_contrib.git
-xcopy /s /r /Y opencv_contrib\modules\optflow\include\opencv2\* C:\opt\opencv\build\include\opencv2
-```
-
-## WGET
-https://eternallybored.org/misc/wget/1.21.4/64/wget.exe download to c:\opt
-Add c:\opt to system env PATH
-
 ## Run Developer Command Prompt for VS 2019
-## Enable Developer mode on in windows system settings
-
-#### Boring SSL - not needed until md5 hash is needed.
-Clone in in c:\opt\
-```
-git clone https://github.com/firebase/boringSSL-SwiftPM.git
-```
+Enable Developer mode on in windows system settings
 
-# Opencl headers
+## Install dependencies
+open cmd.exe and run windows_install_dependencies.bat with target directory (default drive c:\\opt) and force reinstall flag 0/1
```

Comment:
What does the second part mean? `with target directory (default drive c:\\opt) and force reinstall flag 0/1`
How does it relate to below line?